platform/pom.xml

```diff
@@ -159,16 +159,6 @@
             <artifactId>asm</artifactId>
             <version>7.1</version>
         </dependency>
-        <dependency>
```

Comment:
are you sure these can be removed? they are here to prevent conflicts, they are transitive dependencies of other dependencies we use.